equistore/operations/normalize.py

```diff
@@ -0,0 +1,63 @@
+from equistore import TensorBlock, TensorMap
+
+from . import _dispatch
+
+
+def normalize_by_sample(tensor: TensorMap) -> TensorMap:
```

Comment:
let's keep this one out of this PR for now

Reply:
ah ok... we put it in a new branch?